iroha_p2p/src/peer_state.rs

```diff
@@ -0,0 +1,37 @@
+use tokio::time::Instant;
```

Comment:
Doc says that this time is monotonic and not decreasing.
https://docs.rs/tokio/1.7.1/tokio/time/struct.Instant.html

Do we need it here? I mean they might differ a lot between peers. I guess it might be better to use monotonic time. I have no idea about specifics for time in network protocols.

Reply:
It is just to keep track how much time ago was this peer active. Like time of last message in either way, to send occasional pings.

Reply:
So it wont be send within network, right? There are some unconnected parts in the code and sometimes its hard to find relationships between things.

Reply:
You will need this module, right? It is not used anywhere right now

Reply:
Removed.